src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -1062,6 +1064,7 @@ private static ExecutorService getExecutorService() {
     /**
      * Builder to construct a Worker instance.
      */
+    @Getter
```

Comment:
There is no use for the builder to have a getter, we should be testing what the builder has created, if that has the correct value.